webapp/src/main/java/org/apache/atlas/web/errors/ExceptionMapperUtil.java

```diff
@@ -19,15 +19,88 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.List;
 
 public class ExceptionMapperUtil {
     protected static final Logger LOGGER = LoggerFactory.getLogger(ExceptionMapperUtil.class);
 
     @SuppressWarnings("UnusedParameters")
     protected static String formatErrorMessage(long id, Exception exception) {
-        return String.format("There was an error processing your request. It has been logged (ID %016x).", id);
+        if (exception == null) {
+            // If the exception is null, return a minimal error message
+            return "{\n"
+                    + String.format("  \"errorId\": \"%016x\",\n", id)
+                    + "  \"message\": \"No exception provided\",\n"
+                    + "  \"causes\": []\n"
+                    + "}";
```

Comment:
Is this what we generally do in metastore (manually format JSON strings)? Or do we tend to use some object -> serde library for serialization?

Fine if the former, just want to be consistent. (If the latter, I suspect we can simplify a number of these loops by just letting the serialization library handle things.)